internal/extension/oop.go

```diff
@@ -201,8 +195,12 @@ func (p *OOPExtension) monitorStderr(stderr io.ReadCloser, stopChan <-chan struc
 			}
 
 			if scanner.Scan() {
-				lvl, text, err := parseStderr(scanner.Bytes())
-				p.logStderr(lvl, text, err)
+				logLine, err := unmarshalLogEntry(scanner.Bytes())
+				if err != nil {
+					p.logger.Error(err, "failed to parse extension log entry")
+					return
```

Comment:
So in "non dev mode", (i.e. unstructured or regular logging) this would fail?
Do we then say that an extension needs to _always_ log in structured and the parent would "convert" when needed?